crates/hotshot/src/tasks/task_state.rs

```diff
@@ -185,6 +196,22 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> CreateTaskState<TYPES, I>
     async fn create_from(handle: &SystemContextHandle<TYPES, I>) -> ConsensusTaskState<TYPES, I> {
         let consensus = handle.hotshot.consensus();
 
+        let event_stream = handle.internal_event_stream.0.clone();
```

Comment:
What happens during restart? Looks like we'd always have a timeout set for view 1, but if we're rebooting from a higher view this would be wrong. Should we check the view number and only add the task then?

Reply:
Right, this makes sense, let me think about it

Reply:
Updated